README.md

```diff
@@ -27,6 +27,8 @@ To use the Zeth functionality, 3 components are required:
 
 We use 3 terminals, one for each of the above components.
 
+Note: Mac users should increase docker runtime memory from 2gb to 4gb to make Terminal 1 running.
```

Comment:
`make Terminal 1 running` -> `allow Terminal 1 to complete successfully`

Reply:
Changed